scripts/ci/jobs/gke_nongroovy_e2e_tests.py

```diff
@@ -15,6 +15,9 @@
 
 os.environ["ROX_ACTIVE_VULN_MGMT"] = "true"
 
+# delegated scanning support in the secured cluster
+os.environ["SENSOR_SCANNER_SUPPORT"] = "true"
```

Comment:
Initial thought is not necessary because controlling if a scan is performed 'legacy' or not is determined by the delegated scanning config (outside of the OCP internal registry).  This setting configures sensors to be able to perform local scans, but doesn't actually make any scans local.  As it stands we also have the groovy tests which operate with local scanning disabled by default (so already have coverage for the legacy case)

I could see a future benefit to testing the various permutations of env vars and how sensor behaves (such as memory consumption with secrets being stored vs. not) - I could be wrong but doesn't look like we currently have that level of detail/metrics in our tests.

This also could 'technically' be changed within the tests (ie: temporarily setting `ROX_LOCAL_IMAGE_SCANNING_ENABLED` = `false` on Sensor) and doing a sanity check.